.github/workflows/scripts/disable_statements_check.py

```
@@ -0,0 +1,332 @@
+
+
+class DisableStatementsChecker:
+    """Checker for various disable statements in code files."""
+
+    BINARY_EXTENSIONS = {
+        ".png",
+        ".jpg",
+        ".jpeg",
+        ".gif",
+        ".webp",
+    }
```

Ruff (0.15.1)

[warning] Mutable default value for class attribute

(RUF012)